odxtools/templates/macros/printAdminData.xml.jinja2

```diff
@@ -44,6 +44,21 @@
    {%- if doc_revision.tool is not none %}
    <TOOL>{{doc_revision.tool|e}}</TOOL>
    {%- endif %}
+   {%- if doc_revision.company_revision_infos %}
+   <COMPANY-REVISION-INFOS>
```

Comment:
I'll try. This is quite a bit of effort though, because these "drive by" fixes like here can be non-trivial and hard to "unentangle" from the other changes...